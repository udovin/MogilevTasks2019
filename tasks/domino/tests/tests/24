805
13280947 48804
16432148 18030
-5168449 32431
17476281 10434
21162679 9675
691776 2332
-2701489 34309
13002406 52125964
19573352 26381
-5630704 73929
14900332 79510788
17189475 1707
-6500316 49146
9999133 5144
-1826113 4289
6843017 2331
-631783 72309
19651134 3365
8678181 12300
-10377497 970
3988798 3
-8389997 1288
10387649 1462
2292066 13829
-1034585 42208
-4650903 4254
18940203 6085
7570011 58223
-2004917 4791
-4526194 51328
900157 668
14773038 16259
13406709 24295
-3576750 16015
-8109373 16380
-7345636 5707
-1100987 18141
20686648 348
11505380 23312
7829010 37955
4733228 31456
20083086 4375
15399805 47687
-8712963 36953
-9069468 92626007
-6654416 53756423
17440179 9843
-2461398 27517
14426510 32960
4246845 4653
-1565352 158
18606498 18061
6627060 19238
17108360 1631
21378863 54194
7636487 317
11891587 5039
15629140 38714
5576560 9270
20931778 7980
16713321 44886
-3692127 3149
7402045 31756
7362063 26900
7163794 19868
-3696167 16135
2838663 58956
-776709 14173
-2998815 15204
-9873839 18756
-3632286 22532
-1233236 47427
16636189 39387
19794776 912
-8286907 11383
796795 28521
5076456 7502
-11104735 43887
10939936 14070
-6778749 14119
-3600268 1634
-299587 2688
13020266 15999
4336240 970
72042 7203041
5413944 747
17315106 18136
-7591666 13514
13641986 8059
5900457 1052
3016037 25324
15080121 4607
3956151 13925
10850719 655
-142316 4776
-10143230 31675
9393656 29895
14617642 3262
-2845944 14166
13462761 49744
20463523 30592
19470088 10508
9609710 10724
-7538790 7198
11668185 9266
6672142 11579
10160834 5987
21464141 5562
7231326 62524
19739032 16191
17106238 45729
8484869 34251
6501519 48390389
2035749 3772
-236206 3908
3820207 24155
13862394 18957
-10624572 5843
-390026 52553
-9226770 7938
1726605 89
-9477533 45368
-1749536 47995
5347960 3255
16798439 430
-9535806 8371
-5964188 12627
-4501783 23553
12664891 75659
11814070 47707
-3281554 15310
4502681 19275
9814866 17781
7319261 21313
7879976 23664
16044723 24913
-1919658 25639
5715281 25263
-2602442 11628
-5088718 48703
-307099 33424
17554532 2162
-11711611 583
-2221882 57267
4066719 13288
7453860 7936
-10381036 57662
-6460503 21504
3086503 36327
5879759 35193
17185811 78191644
-997236 5008
7046066 3166
-2750913 28181
4147426 4434
-6057987 14845
4289997 2605
-9577223 91411006
7678751 16683
1331761 1378
9491180 26404
138943 19829
-9899433 16332
-1216370 3401
-11724470 1002
-1290761 53728
-7721904 2629
21056871 14488
3223555 4308
11651150 44279
14481218 15044
12382415 2221
20268427 7937
4396666 8955
1927668 18580
11074264 65279
20738475 4479
695256 1654
14990839 2746
3404971 58554
20810351 65294
8878154 3554
5205568 14031
-3433764 22817
-774169 311
11762310 3496
-4781801 31413
-11628537 4636
20385398 28418
12789440 41545
19300915 477
-1566873 9600
21132848 85119397
14582962 25915
-4092651 2387
2041192 842
-6284167 3806
7653457 14053
11699396 423
-6398985 6226
16900173 15256
21581687 18315
-850119 2811
-4495041 2403
13981287 11581
17859719 64936108
15121419 40041
-9353642 4869
20878260 13848
-11807288 28782
-11857505 16200
8083425 11096
10629456 25
7482638 5311
4978104 8603
8163542 27137
9025497 29562
7761177 7728
1849430 55737532
2888676 17437
10698500 46564
2147309 30560
-8874119 35151909
-4143692 23621
574517 6596
6632706 3093
14245183 2883
-2329677 17214
-7578778 5282
18520197 24630
6640247 5103
12228598 46722
18290220 35106
7920495 27097
-2182598 18740
14661464 17455
18985981 33659
4452230 10151
-11930960 777
4176179 15282
19088069 36232
-5430129 15313
-4583329 18701
3338207 20683
-6139432 11390
10293276 84721883
13758637 19229
-795863 17709
21559988 671
15026289 10059
-10857219 14631
16797497 1004
11151121 35263
18808727 29736
-7270290 52120
-8757671 7992
345350 44378
-10507132 50715
18884444 11254
12271921 6656
-10961500 60478
9735952 1466
-7728628 1907
-7212012 31599
11749625 23384
-8814553 51589
6043909 279
1035252 2087
-1358935 265
-12087735 28925
-7815136 38930
3491094 4179
12586465 17855
17938915 11753
12001253 21866
15557955 74954
-5419385 9775
15480868 13762
7326263 3752
3586366 43791
21706757 8455
982248 77536
16595363 30402
4742425 5292
10320739 15301
-6727211 7948
12151267 35244
3467430 37496
-11564452 10667
-3950503 24066
-8673984 18751
11371605 28399
-9767430 23912
-7481894 52546
-2025780 20432
-4341705 33226
16494707 20042
5331440 63328
1396134 2103
15420679 15192
18702389 23989
5895048 8546
8773881 44689
5025639 31046
-10759869 1414
-4217680 25581
15244580 51597
16313484 40488
-10289164 37068
16853903 2680
10054413 43732
21770836 15899
3173791 18399
-3996456 35258
4287203 5915
7484505 16
19342841 32835
-4708250 71765
10125873 32644
19949956 19313
-6569454 13578
14586090 1315
9578251 24393
1575646 28585
208938 3261
-10327360 38785
-6989853 7337233
-6596282 22324
-5980295 18
19391446 17817
6036561 73156
18244614 43393678
13965118 64542
7454279 363
-2291052 38574
17675987 7257
3746589 6622
11595324 36822
16796130 6332967
21233253 24579
13145010 22298
14212865 24896
20151828 4453
8705651 21232
-5916196 25051
4561095 26778
-11305361 41750
9274155 7098
10535047 59917
-5204004 11861
14969653 15196
16000308 5967
14717151 33468
9238141 18277
-86451 1618
14352552 43258773
20223359 34443
10776971 11220
-1517272 65564897
-9236105 9603
-2062264 1127
-7352111 34907
3933903 630
-7925644 64256
20485658 11834
5119069 40195
20123531 24880
-6375801 14874
-584796 7566
-7793326 7827
-11941170 45488
7495129 10088
18173462 10720
11457163 25450
-5275914 571
17536736 29052
-11154585 27009
18071747 12196
17341737 3137
2717021 37969
4982850 4400
-8488670 16604454
9283476 5042
15769680 1131
11942616 58458547
9865285 42698
3265406 14383
-10075503 29484838
13207593 56354
8836554 5109
-8545091 29543
8896676 7350
10843911 48472
-7446833 33087
-11221242 29884
7107556 38724125
18547402 9497
-9128889 12904
17246429 33377
4589524 9011
14583808 587
-11432878 13773
-8004635 43492
15691128 39285
9335858 42201
-2150950 18748
-1438115 6842994
21456174 51234
-6260355 5718
12377654 17099
-8226024 20375
-7089632 1189
8296730 1864
19271269 54439
17038562 24273
-4174255 2398
10234459 36900
16825466 6368
-10815844 16753
-5839144 48988
-11261883 30702
9123793 11537
-4182780 9115
15329705 20551
-90086 20587
-1139285 12812
9197600 32496
-2385663 3350
6236046 67723
-4925113 22071
5640392 10188833
-10041388 20443
16939083 3913
11893986 978
-4270265 7891
9099707 3559
15148509 7689
1491082 9797
-4819546 18025
-10712396 8367
20296044 15264
16555903 9311
2649802 15204
-3280152 413
8433688 43651
6789488 38793
14093803 13314
6577699 235
-6205266 38433
20141100 15355
21017549 26900
-11464937 3194
1597186 19531
19692827 6273
14022686 14953
-3250531 1059
-1823701 1603
-9792268 4342
-12037130 3071
3924469 64685
-3174050 42352
-7112037 25366
-5222469 4243
-8919685 30057938
-1165393 70
-6623598 7769
4432749 400
19416591 186
13701611 18872141
-10885117 4030
-8866870 833
15039104 6675
-12007387 16572
3715504 15179
-3361842 63142
9793102 35433
-8171455 11892
-6309213 34231
-11676303 19145
-215564 3691
17421816 55174
-1362241 23109269
-8590193 30925
2348018 26003
19020888 18537
-9981100 21901
10143233 556
13886094 17126
15015035 9784
2543221 3948
6163275 14569
2452849 60780
6905532 98658178
7049020 1915
-5277738 89047170
-4474105 10948
6341249 24048
1461283 38175
19868360 27149
-10489475 1181
-9051383 11715
-2080592 14771
-9343010 4214
12728607 644
-10561638 55221
-512048 32918
17362744 9581
18009329 174
3646057 44528
5254415 48403
-11590538 37479
2166064 5255
-11743184 9885
6090184 5661
13800621 33572
12860695 32846
6674898 1974
-1850092 42956
-2812796 20447
-4691075 4971
1370277 28471
-8302159 20750
1467429 3698
2206655 10570
-4293123 1375
9732358 8204
-9163460 3783
20347039 1601
-29707 38944
2898814 6513
5488136 46462
2102992 2324
3522431 12029
20947319 6446
6314294 7482
-8072249 7791
-5523939 5803
21541481 11458
8872306 8598
15249663 1590
-5946176 1733
5955101 41627
18290 15030
5565815 50781
13544116 59611
-8449985 14839
1076743 5288
12496302 2627
16543726 40296
-3747224 17505
-2979630 4263
12337850 66414083
-6352266 10960
-2454569 1227
16373203 54611
17782174 33657
11970788 22630
7636098 39958
-3779844 59970
1223481 58217
18659707 66681132
5427384 7077
12923124 45739
1151666 20109
-12156731 2801
11215503 14550
18921485 9989
18739556 26953
-941850 23010
9628763 8922
-7063036 20005
189264 45408
5900726 63
16237510 37956
-4116836 14408
-3593816 4248
17710776 18685
-380591 8712
-5022424 58162
-10461959 24465
-9743186 8161
2274544 53721
1263535 16109
-2868435 85275852
-12160765 3519
21653960 22671
-4038688 40405
-3123156 10207
7282310 50115
896418 4274
-11217693 381
-1624357 46570
294003 21753
5783687 17257
8623231 21052
-9807399 50054
1391902 627
-711163 48439
19674364 10461
-5510988 7330
12460842 65958
16847856 8739
8953551 10154
5835130 26182
20251702 25446
-4397212 10067
11290128 47259
-10648456 42296
19190666 15242
-5814823 3481
448265 16846
-7731717 57353
-7092875 9048
12077790 37872
6442182 50747
20836372 9417
696977 263
6743226 59397
18394661 2607
1771155 25648
-11031935 65558
-2936307 30823
-2172008 1002
-1154606 8336
4361035 10599
8030341 9446
-9683114 58282
-452515 30736
-1580876 36606
-3875574 47747
-3046107 2262
8276041 15840
-856656 8030
11691804 20136
-1837530 12448
-6484669 8911
-5553341 64772423
18134911 42989
-6876788 22372
20572585 46095
18462014 47537
746365 6154
2512376 58792
4950533 72792
11414946 18400
4625059 3619
2775323 25661
-11585380 1223
5141963 502
20646097 22345
17008951 22037
10428353 5415
17614616 47941
-3264881 4475
1029816 13408
14367973 7550
19355470 9701
8237854 32723
14161728 8643
3133712 30317
11114606 20820
1650797 29280
14536015 46520
-8352006 24858
-4434790 5169
19642262 17104
16854442 94
4869555 15109
9945862 38358
2598372 23901
12792396 1048
15818880 6787
-9411917 19440
492396 15639
3246390 8909
10376428 5247703
19451492 29217
898377 459
-1965926 25448
20731923 17599
-1719599 20222
19497355 12654
10629325 15457
18200221 14224
10918500 53121
13605168 34335469
21557821 7407
1987573 8195
21771810 557
-7878019 19265
-7605360 54668916
1524132 1204
6390478 64889171
-6847679 12934
10608085 67072
14037483 8427
14240018 24570
20164839 9440
11548780 6358
-2535787 36583
7979069 46691
-1698081 16924
-7390135 18662
6981734 4294
18413760 4667
-9170337 35259154
15763751 67807
-4410069 10370
5067730 20915
18347839 49344
11869174 3871
-2753 13169
10466114 24690
12548796 70596906
823130 24130
-6907571 1603
-10225721 56126874
-5355189 31879
13336958 34900
2948535 47907
-2361587 23445
13065721 6244
-4887539 1316
-9263357 22149
16564842 3698
3738344 9750
9710328 29015
-3502125 3463
9952571 3900
-3845375 28575
-2457941 962
19166616 8448026
-5764470 15259
411016 63369
-6495446 4531
9546476 12752
2386306 5490
16095994 7804
8518501 22625
-1811338 177
4656954 22829
-1950303 4191
14819855 7813
5140720 5210
18864769 12422
4592112 336
14600717 10467
250334 20238
15922030 27233
10999778 7385
-3338165 21299
11550695 1446
15840408 10170
8599684 17191
1384028 13126
8928739 23422
14277602 25265
-9895410 2653
21168150 657
-7184081 21234
-6632734 19119
685518 63144
4816266 44248
20074989 15602
4513353 2625
704400 2493
-8657508 7663
6583964 4882
-11353946 18646
-5706485 19057
2968138 6221
621772 26102
-150698 38355
15693580 2173
-7658939 9954
20002991 4388
6914620 7934
-7338233 1748
8352612 11216
20502702 7833
9439476 14123
16920440 20231
-2628153 30243
7001153 6725
21304153 84783879
8038163 3996
1905938 24243
13624601 12663
3857905 22310
15173871 1863
-8988127 16771
18844679 19138
-11895079 3479
-9650919 21497
-864911 16469
-10891820 48467020
16163199 32768
12491217 4319
8376499 6605
14622439 1135
19536092 25065
7011858 9976
-11490326 32816
3987827 29351
-4952474 24697005
