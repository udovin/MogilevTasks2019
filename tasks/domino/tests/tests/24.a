1 1 1 1 2 1 1 205 1 1 162 1 4 1 2 1 2 1 1 1 1 1 1 1 2 1 1 1 1 3 1 1 1 1 1 1 1 1 1 1 2 1 2 1 733 678 1 3 1 1 1 1 3 1 1 1 2 1 1 1 1 1 1 1 1 2 3 2 1 1 1 2 1 1 1 2 1 1 1 1 1 1 1 1 510 1 1 2 1 2 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 2 1 1 4 1 2 2 358 1 1 1 1 1 2 1 1 1 2 1 1 1 1 2 2 1 2 2 1 2 1 1 1 1 1 1 2 1 1 3 1 2 4 1 1 4 108 1 2 1 1 1 1 745 1 1 1 1 2 1 1 1 1 1 1 4 1 1 1 1 1 2 1 1 1 1 2 1 1 1 1 1 1 1 1 2 1 2 16 4 1 1 1 1 1 1 1 1 1 1 1 93 2 1 1 1 1 1 1 2 2 1 1 1 466 2 1 2 729 1 1 1 1 1 1 1 1 2 1 1 2 1 1 1 1 1 1 2 1 1 1 265 1 1 1 1 1 2 1 1 1 1 1 3 1 1 1 1 1 2 2 1 1 1 1 1 2 1 1 1 1 2 1 1 1 1 1 3 1 1 1 1 1 2 1 1 2 1 1 2 1 1 1 2 1 1 1 3 1 1 1 1 2 1 1 2 1 2 1 1 2 1 2 3 2 1 1 1 1 2 1 2 684 1 1 1 3 86 4 1 1 1 1 1 122 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 177 3 1 548 2 1 3 1 2 1 3 3 1 1 1 2 1 1 1 1 2 1 1 1 1 1 720 1 1 226 1 1 757 1 1 1 1 2 1 2 340 1 1 1 2 1 1 1 2 1 1 547 2 1 2 1 1 1 2 1 1 1 1 1 2 1 1 2 1 2 1 1 1 1 1 376 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 2 1 1 1 1 2 1 1 1 1 5 1 3 1 730 1 1 1 1 192 1 1 1 1 1 2 2 1 2 1 1 3 546 1 1 1 1 1 1 1 1 1 3 347 1 645 1 1 3 1 1 1 1 1 1 4 1 1 1 1 1 3 1 1 1 1 1 1 5 1 1 4 2 1 1 1 2 1 1 2 1 1 1 1 1 1 1 1 1 2 1 1 1 1 2 1 1 1 1 3 1 1 219 1 1 1 1 1 3 2 2 77 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 2 1 586 1 1 1 1 3 3 1 4 1 1 1 3 1 1 1 1 3 3 1 1 2 1 1 2 1 1 1 3 2 1 1 1 1 2 1 1 1 1 1 1 1 2 2 1 3 2 1 1 2 2 3 1 651 2 1 1 1 1 2 3 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 2 1 1 2 1 1 1 1 1 1 263 2 1 2 2 1 1 2 1 2 195 2 1 1 1 701 1 360 1 3 1 2 1 2 1 1 1 1 1 1 736 3 1 2 2 1 1 1 213 1 1 759 1 1 2 1 1 1 1 1 2 3 1 1 1 1 65 1 2 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 3 1 1 1 1 1 2 6 1 2 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 2 2 1 12 1 2 1 1 1 1 1 1 1 3 775 1 1 1 1 1 1 2 2 638 
