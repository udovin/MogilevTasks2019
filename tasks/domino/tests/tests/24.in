453
-19591577 14848
-37418470 72950
-20389834 187
-19808759 74481119
-30218821 5783
-19226123 32007
-23195452 105238
-50729544 37574
-23379158 19841
-33515477 32334
-46498433 32993
-25730831 6777
-24461276 22917
-28566385 35898
-35954770 23740
-46624148 1095
-35325686 78813
-37997796 62291
-40425707 51803
-27898611 44524
-43726037 67523
-28301532 40389
-35416842 86873
-23767661 12209
-41239847 6305
-24057824 49707088
-33172781 8522
-45371655 68555841
-22584426 51122
-34365445 38889
-35694304 11412
-36431034 95077
-38882144 9895
-26673074 22740886
-25044048 17145
-23360203 3740
-46120490 19580
-33013814 13036
-48613179 123534
-31242354 44049
-21439792 74128075
-41076573 30033
-39119354 18756
-32517870 47287
-20217270 3686
-36886778 2898
-42429058 1815
-30774374 62252
-31130284 498
-44359060 93449
-41912021 10188
-40786512 14105
-23687461 67606
-42068721 59749
-48382120 11013
-35087945 14762
-17229511 115866
-37866587 9804
-32119532 9775
-46849312 12469
-17643676 69631490
-50072270 7356
-19708724 7621
-20957736 3849
-50635315 51668
-32924568 18054
-39150873 9493
-34052230 71371
-44119641 12693
-38937434 42140
-44791300 31004
-28911740 123886
-47966835 20127
-40082544 26031
-38657270 361
-36042299 14690
-18325348 71685
-29375244 32419
-48524284 4405
-29867311 9653
-38185965 107304
-18738346 99086
-35741656 126566
-40629965 8161
-27099312 1011
-34507971 32629
-39621574 77665
-22061835 73548
-21886290 15168
-43567185 86381
-48746713 38016
-17972958 7672315
-22712115 10456
-28104729 37853
-36563187 6106
-28766607 65213915
-43131945 18543
-27518833 39462
-23826153 18763
-50025944 40581
-27106864 66192
-47897494 13071
-42290537 427
-33207187 83096
-31214971 6286
-19952650 3209
-50108085 58296
-39369188 32055
-33699573 92319
-45159586 57100
-41683035 94147
-21131865 81171
-28096301 3657
-28211217 79017
-34147934 9505
-28616150 13113
-46076057 30283
-18877201 121189
-20388007 967
-41427314 228
-17726122 136134
-47421420 8238
-48823576 83467
-18663275 51026
-34999178 16379
-48912586 3695
-44466999 18954
-24430412 11702
-38114874 764
-47776255 15747
-44493027 19320
-30106565 29120
-47282971 8144
-40865114 12054
-21656165 16680
-44108860 10408
-29988036 83943
-26481445 18601
-25625320 102463
-41427744 32661
-21595108 43037
-25119588 104325
-30314370 78837
-42174376 68817
-32478515 20736
-40922013 30179
-41400568 6846
-37034468 22069
-38185431 112
-19181578 32073
-36107650 28625
-49872481 51691
-33454703 37055
-41928745 22662
-18543463 57563
-40500022 3022
-49804339 65858
-32092579 7857
-23483654 73325
-22946558 109466
-33155219 9554
-21760876 99914
-44142807 15946
-41299729 52716
-27200171 101564
-19070154 15212
-22446695 91288562
-43042723 28956
-18052067 18055
-26381003 61792
-28443230 27009
-49984448 39566
-26326596 40183
-20552438 35075810
-19313680 76060677
-36339357 62127407
-47656614 17022
-41546625 100998
-35903588 49688
-42751500 10846
-42430901 46333
-38999836 30821
-46388643 41109
-32837477 10274
-41465647 2174
-40149316 34564
-46299354 46607
-39560256 39151
-22734546 44721441
-18097207 90330
-47357238 30073
-27968863 35095
-22850919 3504
-49294699 119660
-39417320 1654
-45597291 13125196
-38728103 37396
-21333956 12959
-21223568 4750
-32680066 57319
-20501354 42522
-17349268 38957
-29709933 11869
-21996433 39693
-24536068 6951
-34693915 49762646
-36651076 63689
-45246221 48
-38359783 20664
-34258135 10150
-44352190 1986
-43523830 29186
-23809397 8100
-22273728 10467
-23072659 80340
-46989637 120251
-38659646 31901
-49437708 20502
-38391749 7162
-36277264 3626
-17866435 45718
-22855978 62405
-45905817 23765
-42299174 4927
-18416299 71934
-37697509 16307
-41283681 14651
-46226683 52644
-44239166 58830
-43212077 29059
-37677259 14724
-46048007 20326
-29445597 5912
-24543955 12662
-26802296 13087
-35561802 61658
-36800205 1245
-30523856 29217
-33552071 6073
-38122037 18471
-50517514 43497
-29257623 29408
-25487401 111623
-38327108 28534
-30999489 61702
-36815837 69005
-32117521 593
-34856676 11663
-25793723 21044
-27387432 16307
-19005600 8041
-43720344 5302
-45040932 82658
-37983491 7794
-39873846 12454
-47303054 22972
-42804011 54326
-35731014 7733
-45764428 2689
-39411288 3278
-40807984 3255
-31421622 9723
-38519859 6084
-28631233 128244
-44887639 9022
-47522946 113138
-31304534 1668
-24748720 75235521
-33062855 37658
-32015603 58515
-19767399 978
-17471284 41988
-26086069 123441
-33587901 65115
-19635333 59139
-28147710 44846
-42919745 88458
-32784651 45006
-32004169 4479
-26594172 38514
-24416773 4938
-49220646 49766
-31877869 72423
-37345460 72779
-40361087 11853
-23301423 23698
-32766389 3209
-50466611 1032
-38775130 21881
-48116497 25262
-45451592 132958
-30087987 8399
-26914087 7975
-44542114 84618
-22157188 63549131
-28030430 56033
-31670460 15831
-41134389 38311
-27763297 127678
-18537294 488
-43965346 65093
-47989467 6896
-43840040 108725
-44681899 75903
-23972409 25515
-39753763 110462
-20415861 73377
-25376154 66653
-49171924 11317
-31736173 18034
-24993068 2425
-36159935 16202
-38382979 2259
-50606345 19634
-33320276 60108
-23414178 28420
-20813902 12733
-44942609 45416047
-22928843 6064
-29477606 2611
-26997872 4809
-31133118 33845
-37206491 33481
-43509177 13277
-45246691 7987
-40256887 86388
-30556826 68503
-24297491 17876
-20178032 6923
-25977838 80862
-35126762 55490
-30988788 5232
-37546595 106251
-20050443 6462
-50227978 131383
-32158343 16137
-42410860 6404
-49581500 49453
-25463828 14052
-34605221 36332328
-48191422 91702
-37832078 14573
-19388876 30707
-43396521 37241
-25244592 72829
-20682948 7610
-45346818 22971
-24865256 31470
-45949544 32330
-30855988 51302
-42797141 5196
-34197918 30871
-46636979 76683
-24502880 13016
-21513870 35056
-32256358 28859
-32231300 8550
-39235168 33922
-24197346 45989
-42511162 41318
-33819326 26381
-19442820 94754
-30699518 6353
-22307737 62363
-47133673 199
-35224455 56317
-38913066 19414
-40956499 96633
-31227499 10146
-39032507 484
-43257029 30271229
-24048348 464
-46770620 27613
-27561614 98599
-37149871 49537
-17514132 122921
-43664099 10024
-46377052 6015
-18080085 5099
-20068767 7202337
-27325280 10328
-21072804 54813
-32102357 5357
-27704070 1748
-31642655 22354
-47439301 74149
-18222894 27336
-35884515 8701
-41804807 95472
-19559512 16195
-32361355 110315
-49726655 32009
-41469142 23183
-22389905 27604
-40393069 18694
-48417645 82491
-40686589 3902
-29724060 93623
-48296177 28686
-42651186 80840
-42353215 21669
-50364682 3757
-39419319 119260
-49040128 66416
-29055781 26329
-27475433 20570
-19434372 4443
-33120069 32922
-29597261 32462
-47164708 7077
-48529482 77465
-36280893 25197
-32597769 81267
-34779143 28065
-24896885 61535
-50688289 34631
-20242691 139155
-34495090 2523
-39034130 18575
-31503248 64144
-26221083 54509
-48043390 65400
-17108806 436
-19289675 17821
-27021330 5660
-23561169 13341
-40005690 50638
-24527478 1257
-33810441 5757
-43809419 26485
-38629382 12250
-24600172 16710
-36726153 36857
-49676813 22319
-23969422 928
-29189325 24773
-36929599 47164
-24696793 49434
-45739707 1098
-50382393 50675
-47709330 39998
-36588125 55212
-25834834 22277
-45741961 18527
-27950631 15118
-32862665 48566
-23060789 1892
-40812751 8096
-30400217 42785
-33943289 16584
-48397736 19506
-42774802 18595
-20873190 18376
