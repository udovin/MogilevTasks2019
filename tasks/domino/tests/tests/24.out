1 1 1 35 1 1 1 1 2 1 1 1 1 1 1 1 1 2 2 1 3 1 1 1 1 91 1 383 1 1 1 253 1 124 1 1 1 1 3 3 55 1 1 2 1 1 1 1 1 2 1 1 1 1 1 1 1 1 2 1 6 1 1 1 2 1 1 1 2 2 1 1 1 1 1 1 1 1 1 1 4 2 3 1 1 2 2 2 1 3 1 9 1 2 1 152 1 1 2 1 2 1 1 3 1 1 2 1 1 1 1 2 1 4 1 1 2 1 1 1 7 1 2 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 3 1 2 1 1 1 1 1 1 1 1 1 1 1 2 2 1 1 1 2 4 1 1 1 2 3 1 67 1 1 2 1 1 1 46 26 252 1 3 2 1 3 1 2 1 1 1 1 1 70 3 1 2 1 3 1 385 1 1 1 1 1 1 1 1 1 231 3 1 1 1 1 2 1 1 2 1 3 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 2 1 1 3 1 2 2 1 1 1 1 1 1 1 1 1 2 4 1 1 1 1 1 1 3 1 3 1 103 1 2 1 1 2 2 2 3 1 2 1 1 1 2 1 1 1 1 1 1 1 1 384 1 1 3 63 1 1 1 2 1 1 1 2 1 2 1 3 1 1 1 1 1 1 1 1 1 1 377 1 1 1 2 1 1 2 1 2 1 1 1 2 1 1 1 3 1 1 1 1 230 2 1 1 1 1 1 1 1 1 1 1 1 2 1 1 2 1 1 1 1 2 3 1 2 1 1 1 3 1 1 354 1 1 3 1 2 1 1 1 38 1 1 1 1 1 2 1 1 1 1 3 1 2 1 1 3 1 2 1 1 1 1 4 1 1 1 1 1 1 1 2 2 3 1 2 1 3 1 2 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 2 1 2 1 2 1 2 1 2 1 1 2 1 1 
